src/java/org/apache/cassandra/db/partitions/PartitionUpdate.java

```diff
@@ -188,7 +199,18 @@ static PartitionUpdate merge(List<? extends PartitionUpdate> updates)
         return updates.get(0).metadata().partitionUpdateFactory().merge(updates);
     }
 
-    public static SimpleBuilder simpleBuilder(TableMetadata metadata, Object... partitionKeyValues)
+    PartitionUpdate withOnlyPresentColumns();
```

Comment:
For Review:
`withOnlyPresentColumns `, `accumulatedDataSize` and `unsharedHeapSize` are coming from 5.0 before `PartitionUpdate` became an interface. 

The implementations have moved to `BTreePartitionUpdate` and `TriePartitionUpdate`.